cvat-ui/src/components/projects-page/styles.scss

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+@import '../../base.scss';
+
+.cvat-projects-page {
+    padding-top: 15px;
```

Comment:
Please, in all scss files in cvat-ui use $grid-unit-size variable with a multiplier instead of absolute number in pixels. It is related with current design collaboration.
For example in this case it should be: ``padding-top: $grid-unit-size * 2`` is the nearest value to 15.